src/main/java/com/example/demo/Cart/Cart.java

```diff
@@ -0,0 +1,35 @@
+package com.example.demo.Cart;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+
+@Entity
+@Table
+public class Cart {
+    private Integer id;
+    private Iterable quantity;
```

Comment:
Shouldn't cart has multiple products, as far as i can see there is an error in table uml

Reply:
done